pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -2214,9 +2224,11 @@ ParticipantStatusTrackingInfo.Record:
         - RECEIVED
         - UNKNOWN
     requestedAt:
-      type: integer
+      type: string
+      description: ISO-8601 date
     receivedBackAt:
-      type: integer
+      type: string
+      description: ISO-8601 date
```

Comment:
What's the technical reason for this change?

Reply:
That was proposed in the slack channel discussion

Reply:
Can you share a link to the relevant discussion?

Reply:
https://broadinstitute.slack.com/archives/G01S430UNES/p1616680027055800

Reply:
Thank you! How about a different proposal for this- create a new pair of properties: `requestedAtDate`, and `receivedBackAtDate`. These will be the new ISO-8601 date fields. Applications will be able to opt-in to using the new field, and the old one can be deprecated without immediately breaking clients.

Reply:
Sounds good!

Reply:
Rolled back